### PR TITLE
inlcudes all of the assignments from Ruby OOP

### PR DESCRIPTION
**Pull request by jauschalley**
inlcudes all of the assignments from Ruby OOP

### CODE REVIEW

File: Joshua_Alley_Ruby/bank_account/bank_account.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    else
+      "You must choose either checking or saving to deposit your funds"
+    end
+  end
+  def account_info
+    "Account Number= #{@account_number}, Checking: #{@checking}, Saving: #{@saving}"
+  end
+  private
+  def get_acct_number
+    number = 2
```
Comment (nguyenhmp) [inline Joshua_Alley_Ruby/bank_account/bank_account.rb:56]: whats this?

**Comment by nguyenhmp** [comment]
good work and solid code! Keep it up!

**Comment by nguyenhmp** [comment]
no need for returns statements though:)

**Comment by jauschalley** [comment]
Yeah. I figure out the returns bologna when TDD came around. Thanks for
following up.

**Pull request merged**